### DISCUSSION TITLE
Question about project deps

### DISCUSSION THREAD

**Discussion by Mister-Hope** [Mereithhh/vanblog#179] [General]
I am not sure, but this project is not configured as a monorepo with yarn workspace, instead it contain sperated projects while having a root one.

Definitely the outdate learna@v4 is doing nothing in this project.

It's weird to nest node projects, any plan to change it to a valid monorepo, and any plan to replace the classic yarn@v1 with pnpm or yarn berry?

**Comment by Mereithhh** [comment]
You are right.  I've actually been trying to deal with these issues. But i'm lazy, now that you pointed it out, I will improve it as soon as possible.

**Comment by BARMPlus** [comment]
The current project management tool is `yarn`，you can try `yarn workspace + lerna + nx`, if you want, I can help transform

**Reply by Mereithhh** [reply]
in fact, I'm almost done `pnpm + worksapce + lerna +nx`. i will merge to master soon.

**Reply by BARMPlus** [reply]
SGTM👍🏻

**Reply by Mereithhh** [reply]
Basically completed
